### ISSUE TITLE
Can't upload M4V files when "*video*" mimetype is added

### ISSUE THREAD

**Issue by BeckyLiu** [nextcloud/files_accesscontrol#368]
### Steps to reproduce
1.configure file access control flow in administrator page, and add operation block access if " file MIMETYPE does not match '*video*'"
2.upload m4v file
3.

### Expected behaviour
The upload should success

### Actual behaviour
The upload failed with 403 "Access denied"

mp4 files can be uploaded, while m4v files can't with 'Access denied'
After removing the access control workflow, upload m4v file succeeded.
Nextcloud and all apps are updated to latest version.

**Comment by BeckyLiu** [comment]
> 
.*video. gives error as below:
![image](https://github.com/nextcloud/files_accesscontrol/assets/9249061/961b1035-d072-4862-974e-fb00654f02ff)

**Comment by BeckyLiu** [comment]
> Hmm the placeholder is not updating correctly, for the regex we also need the slashes, so this should work: `/.*video.*/i`

`/.*video.*/i` can be saved, but still got 'access denied' when  uploading m4v files

**Comment by nickvergessen** [comment]
Do the same for audio and zip?

**Comment by BeckyLiu** [comment]
> 

audio files, mp4 and zip files can all be uploaded successfully.

**Comment by BeckyLiu** [comment]
> m4v maps to `video/mp4` so that should work: https://github.com/nextcloud/server/blob/ffa7b37d5079dc29ae1ed622c573a3863d118518/resources/config/mimetypemapping.dist.json#L106
> 
> Can you make a final screenshot of your settings?

yes, i checked the mimetype. it should work. but it does not. mp4 files can be uploaded, but not m4v files. 
Configure
![image](https://github.com/nextcloud/files_accesscontrol/assets/9249061/a570d057-6cc9-4bf2-88f9-2f7e08078f13)
